### ISSUE TITLE
Paladin "Seal of Righteousness" is royally screwed up...

### ISSUE THREAD

**Issue by The-Great-Sephiroth** [mangosArchives/databaseZero_Rel19#131]
Alright, this affects all ranks apparently so I will show what my lv12 pally has.

I learned the first rank no problem. Then when I got my Judgement spell, I had a second "Seal of Righteousness" show up that also included the information about what happens when using the judgement spell with it. Both are shown in my spellbook as rank 1. When I got high enough (10 or 12) I got rank 2, which left three versions in my spellbook, one being r1 with no judgement info attached, one being r1 with the judgement info attached, and one being r2.

Now, the first one I learned has the ID 20154. The second one, also r1, has the ID 21084 and is listed as rank 2 when looking up the spell but it is not. The third one that I have, listed as rank 2 in my book, has the ID 20287 and is listed as rank 3 when looked up. Clearly something is not right. I should only have rank 1 in the spellbook but if I delete either one, I lose them all.

To recap:
20154 - R1 in spellbook, R1 in lookup
21084 - R1 in spellbook, R2 in lookup
20287 - R2 in spellbook, R3 in lookup
